src/components/routes/CalendarItemDisplay.vue

```diff
@@ -0,0 +1,241 @@
+            />
+          </svg>
+        </q-icon>
+      </template>
+      <!-- Background: From work -->
+      <template v-if="direction === 'fromWork'">
+        <!-- Icon: From work - active -->
+        <q-icon
+          v-if="active"
+          color="primary"
+          class="full-width full-height absolute-full"
+          data-cy="calendar-item-icon-fromwork-active"
+        >
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 135 38">
+            <path
+              fill-rule="evenodd"
+              d="M126.857 0a8 8 0 0 1 8 8v22a8 8 0 0 1-8 8h-99.2L.857 19l26.8-19h99.2Z"
+              clip-rule="evenodd"
+            />
+          </svg>
+        </q-icon>
+        <!-- Icon: From work - logged -->
+        <q-icon
+          v-else-if="route && route.id"
+          color="secondary"
+          class="full-width full-height absolute-full"
+          style="opacity: 0.4"
+          data-cy="calendar-item-icon-fromwork-logged"
+        >
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 135 38">
```

Comment:
Should be standalone SVG image.

Reply:
Refactored using [public svg with svg:use](https://quasar.dev/vue-components/icon#svg-use-way)
Will be used for new custom SVGs.